test/extended/builds/proxy.go

```diff
@@ -49,10 +49,12 @@ var _ = g.Describe("[Feature:Builds][Slow] builds should support proxies", func(
 				buildLog, err := br.Logs()
 				o.Expect(err).NotTo(o.HaveOccurred())
 				o.Expect(buildLog).NotTo(o.ContainSubstring("clone"))
-				if !strings.Contains(buildLog, `unable to access 'https://github.com/openshift/ruby-hello-world.git/': Failed connect to`) {
+				matched, err := regexp.MatchString(`unable to access.*.ruby-hello.*.Failed.*.connect.*.127`, buildLog)
```

Comment:
Is the 127 bit of this error from an IP address? Trying to determine which bits could be flaky in the future

Reply:
yep from the IP address @adambkaplan

Reply:
An idea dawned on me @adambkaplan after revisiting you point on assessing flakiness.

If I split the build log on a per line basis, and then applied this regexp on each line, it might be a truer test, assuming the regexp can span multiple lines, and get fooled because of that.

thoughts?

Reply:
According to the golang wiki, `.` by default does not match on newlines: https://github.com/google/re2/wiki/Syntax

Reply:
ok @adambkaplan and I discussed in slack and I tried it in golang playground ... the match is not fooled by newlines

Reply:
also see https://github.com/google/re2/wiki/Syntax

but 
```
regexp.MatchString(`unable to access.*.ruby-hello.*.Failed.*.connect.*.127`, "unable to access \n ruby-hello Failed connect 127")
```
returns false

Reply:
bump @adambkaplan .... you still contemplating?

Reply:
@gabemontero so what is the "new" error message that this catches? One of my concerns is that the 127 is an IPv4 address - I'd prefer we future-proof this expression.

Reply:
@adambkaplan comment https://github.com/openshift/builder/pull/99#issuecomment-583125482 from that PR best captures it (though the message has even changed perhaps with one word since that first encounter.

Once this merges, I'll use that PR, where the new version of `git` is being installed, to future-proof it.

Reply:
the run from https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_builder/99/pull-ci-openshift-builder-master-e2e-aws-builds/481 after the last PR in this space merged was where the "slight tweak" occurred.

Reply:
I am amenable to removing the 127 in anticipation of ipv6 if you like @adambkaplan

Reply:
@adambkaplan bump

satisfied with the "future proofing" via https://github.com/openshift/builder/pull/99 ?

final call on `127` ? ... I'm fine either way